modules/singleton-jdk/configure.sh

```diff
@@ -16,4 +16,7 @@ fi
 rpm -e --nodeps $(rpm -qa java-* | grep -v "^java-${JAVA_VERSION}-${JAVA_VENDOR}")
 
 # workaround for https://bugzilla.redhat.com/show_bug.cgi?id=2089626
-alternatives --set java "java-${JAVA_VERSION}-${JAVA_VENDOR}.$(uname -m)"
+_arch="$(uname -m)"
```

Comment:
Any idea why we use `uname -m` here and `uname -i` in the [jdk module configure](https://github.com/jboss-container-images/openjdk/blob/d39eac69f37f4090b89103117fcddbbf24ad680c/modules/jdk/11/configure.sh#L17)? I think we should be consistent throughout.

Reply:
I switched to `-m` because, in one context, I was getting `Unknown` with `-i`. I think that  can happen (at least) in a VM. `-m` seems to be more reliable. If you are happy with that, shall I update https://github.com/jboss-container-images/openjdk/blob/d39eac69f37f4090b89103117fcddbbf24ad680c/modules/jdk/11/configure.sh#L17 in this PR?

Reply:
Yes. We need to use the same. Otherwise we ask for more bugs.

Reply:
OK, I think the `uname -i` = `unknown` problem is a Debian thing; so it is probably safer to align *this* instance with the others, and revisit them all in another PR if I hit the problem again. (I couldn't reproduce it just now in a RHEL VM, or with RHEL containers on a Debian host.)